isic/ingest/models/accession.py

```diff
@@ -46,7 +46,9 @@ class Meta(CreationSortedTimeStampedModel.Meta):
     girder_id = models.CharField(
         blank=True, max_length=24, help_text='The image_id from Girder.', db_index=True
     )
-    upload = models.ForeignKey(Zip, on_delete=models.CASCADE, null=True, related_name='accessions')
+    upload = models.ForeignKey(
```

Comment:
This should be called `zip_upload` too, partly for consistency, but also because single image upload will make things more ambiguous.